docs/guide/src/pcli/transaction.md

```diff
@@ -198,7 +198,7 @@ while read -r i ; do
         echo "Most recently active channel is: $(( i - 1))" ;
         break ;
     fi ;
-done <<< "$(seq 0 10)"
+done <<< "$(seq 0 100)"
```

Comment:
side note, but why don't we just do the real query command? wouldn't that be a comparable effort to documenting and updating the bash script?

Reply:
Totally agree: enumerating all the matches to a query is way more useful cli functionality that spot-checking individual query targets. That's tracked in #2937.